lineage/figures/figure6.py

```diff
@@ -0,0 +1,38 @@
+"""
+This creates Figure 6.
+"""
+from .figureCommon import subplotLabel, getSetup
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import ticker as ticker
+from .Depth_Two_State_Lineage import Depth_Two_State_Lineage
+from ..Analyze import Analyze
+from .Fig_Gen import Lineage_Length, Lineages_per_Population_Figure, KL_per_lineage
+from .Matplot_gen import moving_average
+from ..tHMM_utils import getAccuracy, getAIC
+from ..Lineage_utils import remove_singleton_lineages
+
+
+def makeFigure():
```

Comment:
```suggestion
def makeFigure6():
```

Reply:
Actually, don't think this is recommended. This might break something with the way Dr. Meyer has it set up. Ignore for now.